worker.py

```diff
@@ -251,7 +256,7 @@ def get_status_message(workers, count, start_time, points_stats):
         '',
     ]
     previous = 0
-    for i in range(4, count + 4, 4):
+    for i in range(5, count + 5, 5):
```

Comment:
Oh, I just forgot to remove it, that was just for a screenshot purpose, and I don't think that it would be useful in config for others.